### PR TITLE
Omer's PR for B28G31-92 user story test cases

### PR DESCRIPTION
**Pull request by farukgoc**

### CODE REVIEW

**Review by emmalewiscydeo — APPROVED** [review]
Everything seems good.

File: src/test/resources/features/login.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@login
+@smoke
```
Comment (Micovs) [inline src/test/resources/features/login.feature:1]: What is the logic behind this having a smoke tag?
// I'm curious to test the inline comment, more than anything else...

**Review by Micovs — APPROVED** [review]
Looks good, good to merge

**Review by nurseda916 — COMMENTED** [review]
it's good to go

**Comment by Alexa3462** [comment]
Looks good Omer!

**Review by Alexa3462 — APPROVED** [review]
Looks good Omer!

**Comment by VeraS26** [comment]
Looks good :)

**Pull request merged**